prowler/providers/aws/services/iam/iam_service.py

```diff
@@ -489,3 +516,12 @@ class Certificate(BaseModel):
     id: str
     arn: str
     expiration: datetime
+
+
+class Policy(BaseModel):
```

Comment:
We need to include the tags for the policies right?

Reply:
Done